core/runtime/src/main/java/io/quarkiverse/langchain4j/guardrails/GuardrailsUtils.java

```diff
@@ -0,0 +1,47 @@
+package io.quarkiverse.langchain4j.guardrails;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@ApplicationScoped
+class GuardrailsUtils {
```

Comment:
Should it be named `JsonGuardrailsUtils`?

Reply:
Done.